packages/epics/src/discussions/components/discussion-head.tsx

```diff
@@ -0,0 +1,53 @@
+import { Text } from '@radix-ui/themes';
+import { Skeleton, Image } from '@hypha-platform/ui';
```

Comment:
I'm wondering about this `Text` from `@radix-ui/themes`. Can we get rid of it? What is it's benefit and can't we build it ourself?

Reply:
> I'm wondering about this `Text` from `@radix-ui/themes`. Can we get rid of it? What is it's benefit and can't we build it ourself?

The component was used more for its meaning than for any specific features, I think we can just get rid of it using new typograhpy classes like text-1, text-2 and other